dockstore-client/src/test/resources/arrayOfArrays/arrays.cwl

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+description: "Markdown description text here"
+id: "HelloWorld"
+label: "HelloWorld Tool"
+
+cwlVersion: v1.0
+
+dct:creator:
+  "@id": "http://orcid.org/0000-0003-3566-7705"
+  foaf:name: Peter Amstutz
+  foaf:mbox: "mailto:peter.amstutz@curoverse.com"
+
+requirements:
+  - class: DockerRequirement
+    dockerPull: "quay.io/collaboratory/dockstore-tool-linux-sort"
+  - class: InlineJavascriptRequirement
+
+hints:
+  - class: ResourceRequirement
+    coresMin: 8
+    ramMin: 8092
+    outdirMin: 512000
+    description: "these parameters are used to locate a VM with appropriate resources"
+
+inputs:
+  - id: "#ref_file_1"
```

Comment:
Make sure to add a version uses the map style in V1.0 
see http://www.commonwl.org/v1.0/UserGuide.html#First_example
But leave this as well, we may as well be compatible with both draft 3.0 and v1 if its easy